source/administration/hipchat-migration-guidelines.rst

```diff
@@ -0,0 +1,73 @@
+Migrating from HipChat to Mattermost
+===========================
+
+Migrating from HipChat to Mattermost is made simple with the use of our Bulk Load Tool and comprehensive documentation to help you along the way.  Please use the following guidelines to help you extract, transform and load your data from HipChat to Mattermost. 
+
+Guidelines for Success
+-------------------------------
++ Review the high level steps outlined below in this guide.
++ Review links to in-depth documentation to help you with more detailed instructions on configuration settings.
++ Customize your instance to meet your unique end-user and security requirements. 
++ Contact us at any time to assist you.
+
+Step 1:  Set up your Mattermost Instance
+-------------------------------
+#. Download the `latest version <https://about.mattermost.com/download/>`_ of Mattermost.
+#. `Deploy <https://docs.mattermost.com/guides/administrator.html#installing-mattermost)>`_ Mattermost in your environment using the configuration that meets your organization’s needs for performance and scalability.
+
+
+Step 2:  Export your data from HipChat 
+-------------------------------
+
+HipChat Server / HipChat Data Server:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Atlassian recommends upgrading to the latest version of HipChat Server or HipChat Data Center for access to the Group Export Dashboard.  You can view their instructions on exporting your data `here <https://www.atlassian.com/partnerships/slack/migration>`_.
```

Comment:
Should we also mention this point in this section: https://docs.mattermost.com/administration/migrating.html#migrating-from-hipchat-server-and-hipchat-data-center-to-mattermost

Where we say 
> For teams with large amounts of data, the export function has been reported to fail and it may be difficult to reclaim your team’s data. Consider contacting HipChat support to see if a new solution is available.

Replace "Consider contacting HipChat...." with "Atlassian recommends upgrading..."